packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/package.json.hbs

```diff
@@ -21,6 +21,8 @@
 		{{> tailwindcssDeps}}
 		{{/if}}
         "dotenv": "^16.0.2",
+		"@formatjs/intl-localematcher": "^0.4.0",
```

Comment:
Something is off with the indentation here. I'm seeing both tabs and spaces in the output
```
		"@pantheon-systems/drupal-kit": "4.6.1",
		"@pantheon-systems/nextjs-kit": "1.7.4-canary.0",
        "dotenv": "^16.0.2",
		"@formatjs/intl-localematcher": "^0.4.0",
		"negotiator": "^0.6.3",
		"next": "^13.4.4",
		"next-seo": "^5.15.0",
		"react": "18.2.0",
		"react-dom": "18.2.0",
		"sharp": "^0.31.3"
	},
	"devDependencies": {
		"@types/negotiator": "0.6.1",
        "@types/node": "20.2.5",
        "@types/react": "18.2.9",
        "@types/react-dom": "18.2.4",
		"@testing-library/react": "13.4.0",
		"@vitejs/plugin-react": "^3.0.1",
```
![image](https://github.com/pantheon-systems/decoupled-kit-js/assets/63015754/df5b743f-de62-46b7-a999-f1734798e712)